src/lib/TopLevel.hs

```diff
@@ -335,15 +335,17 @@ evalSourceBlock' mname block = case sbContents block of
           prependImplicit is ty = case is of
             REmpty -> ty
             RNest is' i -> prependImplicit is' $ Pi $ PiType i Pure ty
-  GetNameType v -> do
-    ty <- sourceNameType v
-    logTop $ TextOut $ pprintCanonicalized ty
-  ImportModule moduleName -> importModule moduleName
-  QueryEnv query -> void $ runEnvQuery query $> UnitE
-  ProseBlock _ -> return ()
-  CommentLine  -> return ()
-  EmptyLines   -> return ()
   UnParseable _ s -> throw ParseErr s
+  BadSyntax e -> throwErrs e
```

Comment:
What's the difference between UnParseable and BadSyntax?

Reply:
`BadSyntax` is a failure at the abstract syntax stage.  Comment added to the `SourceBlock'` ADT.